### ISSUE TITLE
Overall Results

### ISSUE THREAD

**Issue by Mohamad73** [hx173149/C3D-tensorflow#72]
Hi,
I am not getting the final result so I am entering the text into excel(WHY!!). However, my results looks very good, I am getting around 90%.

**Comment by SiDinhThoi** [comment]
Ya! the same as me

**Comment by laura-wang** [comment]
yes, same as me, when predict the ucf101 results on model "sports1m_finetuning_ucf101.model", I can get 90.9% accuracy! This is not as reported

**Comment by Mohamad73** [comment]
What could be going wrong? or actually is there anything going wrong?

**Comment by laura-wang** [comment]
I have no idea.
@Mohamad73  And also, do you know what 'c3d_ucf101_finetune_whole_iter_20000_TF.model' is? Which dataset is this model pre-trained on? I can get around 58% accuracy when predict the ucf101 results.

**Comment by Mohamad73** [comment]
I think it is not pre-trained on any big dataset such as Sports-1M. It is only trained on the UCF-101

**Comment by laura-wang** [comment]
Thanks for your answer. The accuracy is quite high, in dutran's paper, train from scratch only achieved ~45% accuracy. It is quite strange, I should check it again.

**Comment by Mohamad73** [comment]
How are you getting the final accuracy? I am not getting it, I have to enter my data into excel to get the result?

**Comment by laura-wang** [comment]
The predict_c3d_ucf101.py outputs file 'predict_ret.txt'. It contains the predict_label and actual label, by processing the txt using python, pandas library, I can then get the result.

**Comment by Ai-is-light** [comment]
@laura-wang I got very low acc, would mid sharing the code you changed for finetune the network. What's more, can you share the sports1m_finetuning_ucf101.model? I can't download it.
Looking forward to any replies.
Thanks

**Comment by BestJuly** [comment]
I meet the same results. If I use the `predict_c3d_ucf101.py`, the accuracy is about 81.23% for the model which is trained from scratch and for the fine-tuned model, the accuracy seems to be 90.54% (89.34% for sports1m_finetuning_ucf101.model). In addition, when I use `Random_clip_valid.py`, the accuracy becomes 83.58% for sports1m_finetuning_ucf101.model. All the results I get seem to be higher than those in the tables. 
I read the codes and it seems to be tested on video clips. The highest clip accuracy is 79.87% and for random-clip, the best result is 76.6% as reported. I can not figure out why. Is it possible that we do the RIGHT thing and just get higher results?

**Comment by xmzhuxm** [comment]
@Mohamad73 @SiDinhThoi @laura-wang @Ai-is-light @BestJuly ，THe results @hx173149 obtained were tested on ucf-101 test split 1,and where your guys trained ,you then split dataset again,so your test.list must have many samples that have been in train _splitt_1(which is used to finetune the pretrained model on sports1m),another way model sports1m_finetuning_ucf101.model have seen your test sample.

**Comment by BestJuly** [comment]
> @Mohamad73 @SiDinhThoi @laura-wang @Ai-is-light @BestJuly ，THe results @hx173149 obtained were tested on ucf-101 test split 1,and where your guys trained ,you then split dataset again,so your test.list must have many samples that have been in train _splitt_1(which is used to finetune the pretrained model on sports1m),another way model sports1m_finetuning_ucf101.model have seen your test sample.   

@malege Thank you for you explanation.
I agree with you for some part. When I use the split01 to train from scratch, the accuracy is very low. Actually, when I got very high results, the training split and the testing split has no overlaps. However, the question still lies to the samples because if we split the dataset by ourselves, e.g. sample id (not the id for categories) 0-4 for training, and id 5-6 for testing, although these 6 videos are different, the scenes in those videos are similar because the same person with the similar background doing the same cation. However, for the split01, there is no overlaps with the persons and the backgrounds in videos. That is why the results are so different.
And when I train from scratch with C3D on split01 training part, when testing, the accuracy is about 30%, and for resnet-18 with 3D convolutions, one researcher reported on CVPR'18 that if trained from scratch, the accuracy is about 42.4%. The result I've got seems more reasonable now.
